repos/system_upgrade/el7toel8/actors/authselectapply/actor.py

```diff
@@ -0,0 +1,50 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.libraries.stdlib import run, CalledProcessError
+from leapp.models import Authselect, AuthselectDecision
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, ApplicationsPhaseTag
+
+
+class AuthselectApply(Actor):
+    """
+    Apply changes suggested by AuthselectScanner.
+
+    If confirmed by admin in AuthselectDecision, call suggested authselect
+    command to configure the system using this tool.
+    """
+
+    name = 'authselect_apply'
+    consumes = (Authselect, AuthselectDecision,)
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ApplicationsPhaseTag)
+
+    def process(self):
+        model = next(self.consume(Authselect))
+        decision = next(self.consume(AuthselectDecision))
+
+        if not decision.confirmed or model.profile is None:
+            return
+
+        command = ['authselect', 'select', '--force', model.profile] + model.features
+
+        try:
+            run(command)
+        except CalledProcessError as err:
+            self.failure(err)
```

Comment:
nit: as you are not reusing code in self.failure and self.success I'd rather not have 2 separate methods but call report_generic functions directly

Reply:
But I won't let this prevent a merge now